lint/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.19
 
 require (
 	github.com/logrusorgru/aurora v2.0.3+incompatible
-	github.com/onflow/cadence v0.40.0
+	github.com/onflow/cadence v0.42.0
 	github.com/onflow/flow-go-sdk v0.41.10
```

Comment:
flow-go-sdk `v0.41.10` is using the older Cadence version (`v0.40.0`). Probably need to update that as well (PR for go-sdk: https://github.com/onflow/flow-go-sdk/pull/488, would need to tag and bump the version)

Reply:
It's still a hassle to update it, given the flow-go/crypto dependency issues, and it should be fine to keep using the old SDK, so I intentionally skipped it

Reply:
Looks like this would be needed for few other PRs as well, so quickly updated the sdk version: https://github.com/onflow/cadence-tools/pull/219/commits/9987c0a442f2405d5040ecc375dc09532b0ee900